assets/src/dashboard/components/cardGridItem/cardTitle.js

```diff
@@ -85,17 +85,28 @@ const CardTitle = ({
     if (!displayDate) {
       return null;
     }
-    return status === STORY_STATUS.PUBLISHED
-      ? sprintf(
-          /* translators: %s: last modified date */
+
+    switch (status) {
+      case STORY_STATUS.PUBLISH:
+        return sprintf(
+          /* translators: %s: published date */
           __('Published %s', 'web-stories'),
           displayDate
-        )
-      : sprintf(
+        );
+      case STORY_STATUS.FUTURE:
```

Comment:
This case is not yet covered by `assets/src/dashboard/components/cardGridItem/test/cardTitle.js`. Seems like an easy addition.